### PR TITLE
no coupling between session and account info query for staking

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Modified the staking query to be independent of each other. This works well on UI

### CODE REVIEW

File: packages/api/src/derives/staking/index.ts [hunk 1]
```diff
@@ -1 +1,11 @@
...
+  queryStakingAccountDetails as accountInfo,
+  querySession as sessionDetails,
+  DerivedStakingInfo,
+  DerivedSessionKeyInfo,
+} from './stakingAccount';
+export * from '@polkadot/api-derive/staking/account';
+export * from '@polkadot/api-derive/staking/controllers';
+export * from '@polkadot/api-derive/staking/electedInfo';
+export * from '@polkadot/api-derive/staking/overview';
+export * from '@polkadot/api-derive/staking/validators';
```
Comment (benxgao) [inline packages/api/src/derives/staking/index.ts:11]: Could we have a general practice guideline on how we are dealing with exporting `polkadot/api` in `cennznet/api`, like whether we encoverage consumers considering `cennznet/api` as the only api interface entry point, or suggest them use polkadot interfaces if possible where we only include cennznet customized api? @holygits

File: packages/api/src/derives/staking/index.ts [hunk 2]
```diff
@@ -1 +1,11 @@
-export {queryStakingAccountDetails as accountInfo, querySession as sessionDetails} from './stakingAccount';
+export {
+  queryStakingAccountDetails as accountInfo,
+  querySession as sessionDetails,
```
Comment (holygits) [inline packages/api/src/derives/staking/index.ts:3]: Nit: a bit more consistency with naming would be good here
```suggestion
  queryStakingAccountInfo as accountInfo,
  querySessionInfo as sessionKeyInfo,
```

File: packages/api/src/derives/staking/index.ts (hunk 1 above)
Comment (holygits) [reply packages/api/src/derives/staking/index.ts:11]: @cennznet/api, users won't import @polkadot/api directly

File: packages/api/src/derives/staking/index.ts (hunk 2 above)
Comment (KarishmaBothara) [reply packages/api/src/derives/staking/index.ts]: done

File: packages/api/src/derives/staking/index.ts (hunk 1 above)
Comment (benxgao) [reply packages/api/src/derives/staking/index.ts:11]: Ok, then lgtm

**Comment by KarishmaBothara** [comment]
> looks good in general.
> Thinking we should move the derived session query to `api.dervie.session.keyInfo` for better discoverability/consistency

Made these changes

**Pull request merged**